src/dune_rules/enabled_if.ml

```diff
@@ -29,6 +30,7 @@ let common_vars ~since =
          match var with
          | "context_name" -> var, (2, 7)
          | "arch_sixtyfour" -> var, (3, 11)
+         | "env" -> var, (1, 14)
```

Comment:
Why is this since version 1.14? Given it didn't work before, shouldn't it be 3.17 or as a bugfix to a feature in 3.15 be `(3, 15)`?

Reply:
This is related to `env` itself. Like `context_name` start from 2.7.

Reply:
surely we need a backport on 3.15 after.

Reply:
I don't understand why this is 1.14 either

Reply:
My bad.  I've got a misunderstanding here, reading the error message. I was thinking it was specific to the variable itself. The `1.14` is coming from the doc, when `env` is introduced. 

```
Error: %{env:--} is only available since version <version> of the dune
  language. Please update your dune-project file to have (lang dune <version>).
```

 Sorry @Leonidas-from-XIV you're right.

Reply:
I add on `3.17`.